frontend/src/components/studio/sidebar/sidebar.tsx

```diff
@@ -0,0 +1,40 @@
+import { FC } from 'common/types/types';
+import { Icon } from '../../common/icon';
+import { IconName, IconColor, AppRoute } from '../../../common/enums/enums';
+import { Link } from '../../common/common';
+import { useState } from 'react';
+import { useNavigate } from 'hooks/hooks';
+
+import styles from './styles.module.scss';
+
+type Props = Record<string, unknown>;
+
+const arrayButtons = [IconName.HOME, IconName.ANALYTICS];
+
+const StudioSidebar: FC<Props> = () => {
+  const navigate = useNavigate();
+  const [activeButton, setActive] = useState<string>(IconName.HOME);
```

Comment:
You do not actually need this extra state. 

You solution will not work if user will open the link for page that different with Home. Get current path from the URL and use it for defining what sidebar option is active.